grub.d/08_mender_rollback.cfg

```diff
@@ -0,0 +1,14 @@
+if [ "${upgrade_available}" = "1" ]; then
+    menuentry "Roll back latest upgrade" --id mender_rollback {
+        reboot
+    }
+
+    # Ideally we would have liked to use a string in this "fallback" variable,
+    # but because there is a bug in GRUB which accepts only indexes for
+    # "fallback", we need to put the menuentry here in the first position, and
+    # then use index 0 in "fallback". In a later script we will set "default",
+    # which does accept strings, so that we don't boot the fallback by default.
```

Comment:
Exactly, the ID does not work.

The GRUB docs [explicitly say](https://salsa.debian.org/grub-team/grub/-/blob/master/util/grub.d/README) that boot entries start at 10, so 08 should be fine. We don't want it too early, because there should be room for adding initialization scripts before it. In the end the choice of 08 is somewhat arbitrary of course, but seems like a reasonable compromise.